parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -147,6 +147,23 @@ installCredentalProvider() {
     rm -rf ${CREDENTIAL_PROVIDER_DOWNLOAD_DIR}
 }
 
+# TODO (alburgess) have oras version managed by dependant or Renovate
+installOras() {
+    ORAS_DOWNLOAD_DIR=${1}
+    ORAS_DOWNLOAD_URL=${2}
+    ORAS_VERSION=${3}
+
+    mkdir -p $ORAS_DOWNLOAD_DIR
+
+    echo "Installing Oras version $ORAS_VERSION..."
+    ORAS_TMP=${ORAS_DOWNLOAD_URL##*/} # Use bash builtin ## to remove all chars ("*") up to the final "/"
```

Comment:
is this really needed. ?  I feel this was for the cotnainerImages with the hacked `:*` at the end